setup.py

```diff
@@ -10,12 +10,11 @@
 
 install_requires = [
     'sqlparse >= 0.3.0',
-    'google-cloud >= 0.34.0',
     'google-cloud-spanner >= 1.8.0',
 ]
 
 setup(
-        name='django-spanner',
+        name='django-google-spanner',
         # Duplicate version here rather than using
```

Comment:
Please single-source the version. You can treat the version in this file as the source of truth and get the version using `pkg_resources.get_distribution()` in the codebase.

[example in python-spanner](https://github.com/googleapis/python-spanner/blob/55a21d97d0c863cbbbb2d973b6faa4aeba8e38bb/google/cloud/spanner_v1/__init__.py#L19)

Reply:
All done.